WalletWasabi.Backend/wwwroot/about.html

```diff
@@ -0,0 +1,258 @@
+<!doctype html>
+<html lang="en">
+
+<head>
+    <!-- Required meta tags -->
+    <meta charset="utf-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+    <meta name="title" content="Wasabi Wallet - Bitcoin privacy wallet with built-in CoinJoin">
+    <meta name="description" content="Wasabi is an open-source, non-custodial, privacy-focused Bitcoin wallet for Desktop, that implements trustless CoinJoin over the Tor anonymity network.">
```

Comment:
All the meta tags must be in the nginx config because they repeat in every single file.

Reply:
Now, it's only copy / paste I will work on them.